data/crac-io/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracImporters.java

```diff
@@ -29,6 +35,40 @@
     private CracImporters() {
     }
 
+    public static void cracAliasesUtil(Crac crac, Network network) {
```

Comment:
The name of this function is not very well chosen :) maybe addAliasesToCrac would be better?

Reply:
@murgeyseb what do you think? I don't have an opinion about this...